src/Compilers/Server/VBCSCompilerTests/CompilerServerTests.cs

```diff
@@ -198,11 +198,7 @@ private static (T Result, string Output) UseTextWriter<T>(Encoding encoding, Fun
             var client = ServerUtil.CreateBuildClient(language);
             client.TimeoutOverride = Timeout.Infinite;
 
-#if NET461
-            var sdkDir = RuntimeEnvironment.GetRuntimeDirectory();
-#else
-            string sdkDir = null;
-#endif
+            var sdkDir = ServerUtil.DefaultSdkDirectory;
```

Comment:
>ServerUtil.DefaultSdkDirectory [](start = 25, length = 30)

Why not BuildClient.GetSystemSdkDirectory?

Reply:
`ServerUtil.DefaultSdkDirectory` is already an existing API in tests, used elsewhere in test code. So, the reason is consistency.

Reply:
Perhaps we should rename ```ServerUtil.DefaultSdkDirectory``` to use consistent naming.